OsmAnd/res/values/strings.xml

```diff
@@ -318,33 +318,33 @@
     <string name="magnetic_bearing_widget_desc">Angle between magnetic north and the target point observed from your location.</string>
     <string name="time_to_go_desc">Time in minutes or hours needed to reach your destination.</string>
     <string name="arrival_time_widget_desc">Time of arrival at the destination point.</string>
-    <string name="lanes_widgets_desc">Shows the current road lanes layout and highlights the ones to take while navigating.</string>
-    <string name="map_markers_bar_widget_desc">The top bar widget displays the distance and direction to the next Map marker from your current location. The widget can be configured to show 1 or 2 markers.</string>
-    <string name="street_name_widget_desc">The Street name widget shows the current street name with a straight location arrow, or the next street name with an arrow indicating an upcoming turn.</string>
-    <string name="coordinates_widget_desc">Shows the geographic coordinates of current geolocation.</string>
+    <string name="lanes_widgets_desc">Highlights the road lane to take while navigating and shows the layout.</string>
+    <string name="map_markers_bar_widget_desc">On the top bar widget you could see the distance and the direction to the marker from current location. It is possible to show 1 or 2 top markers.</string>
+    <string name="street_name_widget_desc">Shows the current street name with a straight location arrow or next street name with the maneuver to complete.</string>
+    <string name="coordinates_widget_desc">It shows the geographic coordinates of current geo-location.</string>
     <string name="shared_string_widget">Widget</string>
```

Comment:
Originals of these 3 lines sound more streamlined?

Reply:
I tried to do better. The more human-like descriptions help understanding?
Gets very technical as it is.